pages/info/info.py

```diff
@@ -7,49 +7,151 @@
 
 layout = dbc.Container(
     [
-        dcc.Markdown(
-            """
-
-            ### Defintions
-            -------
-            __Contribution__
-                > Definition for item here.
-
-            __Contributor__
-                > Definition for item here.
-
-            __Active Contributor__
-                > Definition for item here.
-
-            __Drifting Contributor__
```

Comment:
I see the mapping between the different definition wording used below and these. I think it is important to have formal definitions to the exact terms used/measured in the graphs. This might be a part of the qualitative vs. quantitative breakdown I describe below

Reply:
I'm not sure I 100% understand what you mean- could you give an example?